app/controllers/amusement_parks_controller.rb

```diff
@@ -0,0 +1,7 @@
+class AmusementParksController < ApplicationController
+  def show
+    @park = AmusementPark.find(params[:id])
+    @mechanics = @park.mechanics.distinct
+    @rides = @park.ride_exp
+  end
```

Comment:
This will be totally fine. But I would maybe just send over one object to the view rather than three objects. I say this because you can get `mechanics` and `rides` off of the `park` object. Then, in the view, you can call `.mechanics` or `.park.ride_exp` on the `@park` object that you send over.

Reply:
Looking back now ... Regarding your comment on your PR about the `distinct` in the view. I agree. This could be a good instance of how you could use an instance method on park to get distinct mechanics. You can always call those methods from your view, we just want that logic to live in your models.